README.md

```diff
@@ -69,108 +69,116 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-  - Fix fix for object reference not set.
-  - Fix for encoding UTF-8.
-  - Fix for file is in use. 
-  - Fix for saying the file is too big.
-
-### Image Resizer
-
-- Fix for `invalid operation` error. [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
+- Additional markdown file extensions added for Markdown Preview. Thanks [@skycommand](https://github.com/skycommand)!
+- Svgs are now registered as a picture kind on Windows.
+- Added a text wrapping setting and copy context menu to dev file preview. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Pdf file preview and thumbnails are now disabled by default, due to incompatibility with Outlook and an appropriate warning is shown in Settings.
 
 ### Mouse utility
 
-- Find my mouse will now activate via shaking the mouse with a settings change!
-- Find my mouse now can excludes apps.
-- Fix for crosshair thickness looking odd due to anti-aliasing
-- Fix for a hard crash on crosshair (This was a hotfix for 0.55)
-
-### PowerRename
-
-- Will not spells check file names anymore. Thanks [@niels9001](https://github.com/niels9001)
-- Autocomplete result box to not obscure apply button. Thanks [@niels9001](https://github.com/niels9001)
-- Regex fix
+- Find my mouse has a new setting to specify a minimum distance for activation. (This was a hotfix for 0.56)
+- Fix for the bug causing the task bar to be hidden behind other windows when a mouse utility was active.
+- Fix for the bug causing shortcuts set on icons to not activate when a mouse utility was active.
+- Fixed a slight offset in Mouse Pointer Crosshairs when it's configured with an odd thickness.
 
 ### PowerToys Run
 
-- Timezone conversion plugin for PowerToys Run!  Thanks [@TobiasSekan](https://github.com/TobiasSekan)
-- Hexadecimal and binary numbers now are supported in the calculator plugin. This was added a bit ago and we'd like to extend a belated thanks to [@gsuberland](https://github.com/gsuberland)
-- Terminal plugin performance boost.  Thanks [@htcfreek](https://github.com/htcfreek)!
-- Terminal will now be found via the Program plugin again.  
-- Shutdown command is now using hybrid fast argument for shutting down
-- Support for VSCodium with VS Code workplace plugin. Thanks [@makeProjectGreatAgain](https://github.com/makeProjectGreatAgain)
+- New plugin for time and date values/information. Thanks [@htcfreek](https://github.com/htcfreek)!
+- WindowWalker now has a tooltip, additional features like killing the process and closing the window, additional settings and improvements. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Unit converter now accepts additional ways of writing feet and gallon. Also interprets as either imperial or US gallons depending on current culture. Thanks [@yifan-k](https://github.com/yifan-k)!
+- Unit converter now accepts metre for meter.
+- Localization for Web Search and Unit Converter has been enabled.
```

Comment:
Let's not mention unit converter here to avoid confusion. The units still not localized.

Reply:
I'll add a mention it doesn't include units.

Reply:
Thanks for the review. I've added the mention. PTAL